app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+class Animal(object):
+    alive = []
+
+    def __init__(self, name: str, *args: int) -> None:
```

Comment:
it is wrong solution. It must accept 3 arguments:
name: str, hidden: bool, heath: int
But hidden and heath must have default argument